resources/user-agents/browsers/nokia-browser/nokia-browser.json

```diff
@@ -6,7 +6,7 @@
   "userAgents": [
     {
       "userAgent": "Nokia Browser",
```

Comment:
I added a lot of devices to this file, and also broke the patterns up a bit so that the proper platform + engine was showing based on research I did looking into which phones were on which version of the Nokia OS (Series30/Series40/Symbian).